testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/ExpectedDeprecationWarning.java

```diff
@@ -29,6 +29,9 @@
  */
 public abstract class ExpectedDeprecationWarning {
 
+    private static final Pattern DEPRECATION_WARNING_LOG_PREFIX_PATTERN =
+        Pattern.compile("^.* " + Pattern.quote("[WARN] [org.gradle.internal.featurelifecycle.LoggingDeprecatedFeatureHandler] "));
```

Comment:
💭 Can we reference the actual class to get the name here, in case of future refactoring?